SwiftWisdomTests/Rx/Rx+DelayElementsTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Rx+DelayElementsTests.swift
+//  SwiftWisdom
+//
+//  Created by Son Le on 1/30/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+@testable import SwiftWisdom
+
+final class RxDelayElementsTests: XCTestCase {
+
+    func testDelayingElements() {
```

Comment:
Could you also add a test of the same sequence with the predicate ($0 % 2 == 0)

It should expect `[1,3,0,2,4]`